Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/CustomFilePickerFragment.java

```diff
@@ -41,4 +44,33 @@ public Uri toUri(@NonNull final File file)
       cancel.setVisibility(View.GONE);
     }
   }
+
+  @Override
+  protected boolean isItemVisible(final File file)
```

Comment:
Should `file` here and `filename` in fileExtension also be annotated with `@NonNull` ?